### DISCUSSION TITLE
data-testid

### DISCUSSION THREAD

**Discussion by shubh24i** [elrumordelaluz/reactour#502] [General]
Hi, Can anyone know how to pass data-testid in tour popup

**Comment by elrumordelaluz** [comment]
Hi @shubh24i, should `data-testid` be dynamic? There is a partial implementation to pass directly props into _Popover_ from the _Provider_ props but remains static if the _Tour_ is controlled internally. Otherwise you could pass whatever you want through [ContentComponent](https://github.com/elrumordelaluz/reactour/tree/main/packages/tour#contentcomponent-componenttypepopovercontentprops).
